packages-web/fieldset/src/components/Fieldset.tsx

```diff
@@ -0,0 +1,24 @@
+import { createElement, CSSProperties, FunctionComponent } from "react";
+
+export interface FieldsetProps {
+    name?: string;
+    className: string;
+    style?: CSSProperties;
+    tabIndex?: number;
+    legend?: string;
+}
+
+export const Fieldset: FunctionComponent<FieldsetProps> = (props): JSX.Element | null => {
+    const { legend, name, tabIndex, style, className, children } = props;
+
+    if (!children) {
```

Comment:
If content is required than create a `.editorConfig.ts` file and do consistency errors.

Reply:
This is the inner component. In the xml, the widgets property is required, so we always get children here. I think it's fine to leave the extra check here, to maintain loosely coupled code.

Reply:
Does it throe error if nothing be dragged inside?

Reply:
Yes

Reply:
As this will never be empty, it will never return null. So you can remove the if and also the return type `| null`